addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -92,6 +93,7 @@ FormRenderer.include({
      * @returns {Object}
      */
     _makeChatterContainerProps() {
+        const hasAttachmentUpload = new Domain(this.chatterFields.hasAttachmentUpload).compute(this.state.evalContext);
```

Comment:
Do we want this? And if so, do we really want this to be a chatter option? Isn't it strictly model dependent?